ext/drsyscall/drsyscall_record.h

```diff
@@ -90,7 +90,7 @@ typedef enum {
  * value.
  */
 START_PACKED_STRUCTURE
```

Comment:
It seems weird to have a packed structure that is aligned. Packing is used for 2 things I guess: removing padding (but you're adding it right back) and canonicalizing the size across platforms. I suppose the latter is the reason to pack. This deserves a comment as I don't know that I have ever seen a struct that was both packed and aligned.

Reply:
Are you sure this combination is well-defined? Could some compiler have the packed override the align?